quarkus-workshop-super-heroes/docs/src/docs/asciidoc/0-introduction/introduction-installing-ai.adoc

```diff
@@ -0,0 +1,68 @@
+[[introduction-installing-ai]]
+
+= OpenAI or AzureAI Subscriptions
+
+The Narration microservice needs to access an AI service to generate the text narrating the fight.
+You can choose between OpenAI or AzureAI.
+
+== OpenAI
+
+TODO
+TODO
+TODO
+TODO
```

Comment:
Can we resolve the TODOs before merge? Sometimes people who we don't even know are using the workshop run it, sometimes for clients ... so we should aim to keep it 'working' at all times.